sphinx/tutorial/rose/furthertopics/rose-stem.rst

```diff
@@ -0,0 +1,309 @@
+.. include:: ../../../hyperlinks.rst
+   :start-line: 1
+
+rose stem
+=========
+
+.. warning::
+
+   Before proceeding you should already be familiar with the ``rose stem``
+   part of the user guide.
+
+   .. TODO - link in the rose user guide page when translated.
+
+.. image:: http://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Cassini_Saturn_Orbit_Insertion.jpg/320px-Cassini_Saturn_Orbit_Insertion.jpg
+   :align: right
+   :alt: Artist's Impression of Cassini entering Saturn orbit
+   :width: 300px
+
+This tutorial will walk you through creating a simple example of the
+``rose stem`` testing system which will involve piloting a spaceship
+through space.
+
+
+Getting started
```

Comment:
To maintain consistency with the rest of the tutorial we should capitalise all of the section headings on this page - this has been done for all other pages now but this one seems to have been missed (or written after those modifications were made).